### PR TITLE
Add mushroom ketchup, fried pickled veggies and krampouezh

### PR DESCRIPTION
**Pull request by BalthazarArgall**
#### Summary
Content "Add mushroom ketchup, fried pickled veggies and krampouezh"

#### Purpose of change
Ro din ur c'hrampouezh! We have pancakes yet not their ancestors, also, what if I want to roleplay a Breton?

According to reddit we also lack mushroom ketchup and fried pickles.

#### Describe the solution
I added the Breton delicacy known by most people as "crepes" to the game, there is a sweet and a not-sweet version along with their recipes.

I also added the mushroom ketchup and a semi-ambitious recipe for frying pickled vegetables, not just pickles!

#### Testing
To be done.

#### Additional context
This PR was originally meant to only add krampouezh but I also ran a poll of some kind on reddit last week asking for ideas to add to the game, hence why I would like to thank and credit u/ColeWalkeroftheRim for the fried pickles idea and u/JeveGreen for the mushroom ketchup idea.

### CODE REVIEW

**Comment by BalthazarArgall** [comment]
> You probably want to actually describe what krampouzeh is in its description. "Breton delicacy" isn't very informative.

I 100% intend to change the description, I ended my cdda work session on this very description and commited before reverting it to TODO or something.

**Comment by fairyarmadillo** [comment]
> > You probably want to actually describe what krampouzeh is in its description. "Breton delicacy" isn't very informative.
> 
> I 100% intend to change the description, I ended my cdda work session on this very description and commited before reverting it to TODO or something.

It's a draft, my bad for reviewing it before it was ready.

File: tools/spell_checker/dictionary.txt [hunk 1]
```diff
@@ -3177,6 +3178,7 @@ kPa
 krabgek
 krabgeks
 Krag
+Krampouezh
```
Comment (Qrox) [inline tools/spell_checker/dictionary.txt:3181]:
```suggestion
krampouezh
```
It is not a proper noun so it should be uncapitalized in the dictionary.

Comment (BalthazarArgall) [reply tools/spell_checker/dictionary.txt:3181]: I'm a bit new to this dictionary stuff, thanks

**Pull request merged**